src/components/Categories.js

```diff
@@ -0,0 +1,17 @@
+import { useDispatch } from 'react-redux/es/exports';
+import { checkStatus } from '../redux/categories/categories';
+
+function Categories() {
+  const dispatch = useDispatch();
+
+  const handleClick = () => {
+    dispatch(checkStatus());
+  };
+  return (
+    <>
+      <button type="button" onClick={handleClick} className="status">Check Status</button>
```

Comment:
- [ ] After clicking the Check Status button, please make sure the `Under Construction` text returned from the action dispatched is displayed here.

https://user-images.githubusercontent.com/52934128/185102258-7f6dcedf-97a1-42d6-8ad0-2b4ad87b312d.mp4

Reply:
@topeogunleye Thank you so much ,i have done just that